client/src/components/FeedbackForm.js

```diff
@@ -2,22 +2,38 @@
 import React, { useState } from "react";
 import "./feedbackForm.css";
 
-const FeedbackForm = ({ student }) => {
-	console.log("student",student);
-	const [feedbackMessage, setFeedbackMessage] = useState("");
+const FeedbackForm = () => {
+	const [feedbackMessage, setFeedbackMessage] = useState({
+		feedback: "",
+		name: "",
```

Comment:
@tayfuntayfun do we need to add the name a property here? Or maybe we could find a way to pull it from the DB based on who is giving it?

Reply:
We have to, otherwise you can not get data from multiple inputs unless you know another way of it. This is for posting purposes not for retrieving the data.

Reply:
I am not sure because I do not know much about backend but, I think even for the sorting option we will need the name, date, and maybe modules? to be able to sort the feedback by those parameters...

Reply:
It is right Ola, we now need Module and Date property fort this object.